deploy/kubernetes/metricbeat/README.md

```diff
@@ -2,30 +2,145 @@
+| ELASTICSEARCH_PASSWORD | changeme      | Elasticsearch password               |
+| ELASTIC_CLOUD_ID       |               | Elastic cloud ID                     |
+| ELASTIC_CLOUD_AUTH     |               | Elastic cloud auth                   |
 
-### Settings
+- [metricbeat-role-binding.yaml](metricbeat-role-binding.yaml),[metricbeat-role.yaml](metricbeat-role.yaml) and [metricbeat-service-account.yaml](metricbeat-service-account.yaml) to define a user's permissions in our namespace (more information on this can be found [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)). Notice that the namespace we are using is `kube-system`, but this can be changed by updating the YAML manifests.
+
+### Example
+
+In this example, we will use `kind` to run a local Kubernetes cluster. Our cluster will have more than one node.
+This way we can see the usage of our Metricbeat DaemonSet.
+
+
+#### Prerequisites
+
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/) to run commands against Kubernetes clusters.
+- [For option 1](#Option-1.-Run-locally-on-kind): [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) for running local Kubernetes clusters.
+- [For option 2](#Option-2.-Run-on-GKE): [gcloud Cli](https://cloud.google.com/sdk/docs/install).
+
+#### Option 1. Run locally on kind
+
+The first thing we need to do is create a Kubernetes cluster. On kind, the default [configuration](https://kind.sigs.k8s.io/docs/user/configuration/) of a cluster generates only one node. As we are using a DaemonSet to run our Metricbeat, we will make a cluster run 4 nodes, so we can check our pods are running on all (or some) of them. To do this, we simply create the file `cluster-config.yaml`:
+```YAML
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+- role: worker
+```
+
+After that, we can create a cluster using the command below.
+
+```
+$ kind create cluster --config=cluster-config.yaml
+```
+
+Notice that our cluster was created using the configuration we created above. The name of our cluster is the default one, `kind`. To interact with the cluster, you only need to specify the cluster name as a context in kubectl:
+
+```
+$ kubectl cluster-info --context kind-kind
+```
+
+
+#### Option 2. Run on GKE
+
+1. Create a GKE cluster. You can find information on how to do that [here](https://cloud.google.com/kubernetes-engine/docs/deploy-app-cluster).
+
+2. Connect to the newly created cluster. You can do that by running the following:
+```
+$ gcloud container clusters get-credentials <cluster name> --zone <zone> --project <project name>
+```
+
+
+#### Run Metricbeat on Kubernetes
+
+You can check the nodes of the newly created cluster by running the following:
+
+```
+$ kubectl get nodes
+```
+
+Metricbeat gets some metrics from [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics). You can download the project. You can run, for example:
```

Comment:
Can you please explain this concept a bit better?
I would suggest that we rephrase this to be...

`you can download the project` => `git clone the repo locally and then run kubectl apply....`